### PR TITLE
Feat/confirm message

### PR DESCRIPTION
**Pull request by NadjaKanunnikow**

### CODE REVIEW

File: components/ConfirmModal.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  border-radius: 0.5rem;
+  border: solid var(--primary-color) 1px;
+`;
+
+const StyledModalMessage = styled.h3`
+  color: var(--primary-color);
+  text-align: center;
+`;
+
+const SytledButtonContainer = styled.div`
```
Comment (willywdev) [inline components/ConfirmModal.js:79]: A small typo :)

File: components/StyledButtonOrLink.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
```
Comment (willywdev) [inline components/StyledButtonOrLink.js:1]: Do you need css here?

**Review by willywdev — APPROVED** [review]
Great work! You are writing very clean code!
Since the things i marked are non-trivial, i'm approving this code review.
Please change these small things 🐈

Greetings 🚀

**Comment by NadjaKanunnikow** [comment]
Hey Willy,

thanks for your review! In fact the code was heavily inspired by yours. 🙏 
So if its clean, you know where it comes from. 😉

**Pull request merged**